.landscape.yml

```diff
@@ -0,0 +1,13 @@
+ignore-paths:
+  - doc
+
+pep8:
+  disable:
+    - E133
+    - E226
+    - E228
+    - N802
+    - N803
+  options:
+    max-line-length: 110
```

Comment:
Yes it is: https://developer.lsst.io/coding/python_style_guide.html#line-length-must-be-less-than-or-equal-to-110-columns